packages/forms/src/index.tsx

```diff
@@ -95,8 +95,8 @@ const inputTagProps = <T extends InputTagProps>(
   const {
     errorClassName,
     errorStyle,
-    dataType,
-    transformValue,
+    dataType, // eslint-disable-line @typescript-eslint/no-unused-vars
+    transformValue, // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
I think you can prefix these with underscore and they should no longer throw errors.

```suggestion
    _dataType,
    _transformValue,
```

Reply:
You mean like this?

```ts
  const {
    errorClassName,
    errorStyle,
    dataType: _dataType,
    transformValue: _tranformValue,
    ...tagProps
  } = props
  ```

That still gives me the warnings

```terminal
C:\Users\tobbe\dev\redwood\redwood\packages\forms\src\index.tsx
   61:40  warning  Unexpected any. Specify a different type             @typescript-eslint/no-explicit-any
   98:15  warning  '_dataType' is assigned a value but never used       @typescript-eslint/no-unused-vars
   99:21  warning  '_tranformValue' is assigned a value but never used  @typescript-eslint/no-unused-vars
```

Reply:
Oh sorry, I hadn't read the code properly. Your method is fine.